scripts/update-app-secrets.sh

```diff
@@ -1,52 +0,0 @@
-#!/bin/bash
```

Comment:
Why did you remove the file? Isn't (shouldn't) it used for MAUI demo?

Reply:
Yes, it MAUI is also mentioned in the script but I'm not sure was even used for MAUI. I couldn't find any usage of the script for the MAUI demo app. I could leave the script and remove all xamarin related stuff?

Reply:
It should update secrets. See https://github.com/microsoft/appcenter-sdk-dotnet/blob/56f71941990906039841d50257cdad302e7f169b/Apps/Contoso.MAUI.Demo/App.xaml.cs#L118

Reply:
I've reverted it and refactored it a bit since we've removed xamarin apps